packages/framework/block-std/src/selection/manager.ts

```diff
@@ -177,24 +177,28 @@ export class SelectionManager {
     }
   }
 
-  find<T extends BlockSuite.SelectionType>(type: T) {
-    const computedFind = computed(() =>
+  find$<T extends BlockSuite.SelectionType>(type: T) {
+    return computed(() =>
       this.value.find((sel): sel is BlockSuite.SelectionInstance[T] =>
         sel.is(type)
       )
     );
+  }
 
-    return computedFind.value;
+  find<T extends BlockSuite.SelectionType>(type: T) {
+    return this.find$(type).value;
   }
 
-  filter<T extends BlockSuite.SelectionType>(type: T) {
-    const computedFilter = computed(() =>
+  filter$<T extends BlockSuite.SelectionType>(type: T) {
```

Comment:
Will this naming convention become a code standard in the future?

Reply:
Not really, if we decide to move to signal. We will use signal compute to replace the old `filter` and `find`.

Reply:
LGTM, do we need to mark the old method as deprecated?

Reply:
Not yet, I'm still not sure if we will use signal in the end. Let's discuss in dev committee.

Reply:
OK